src/app/ffbe/services/ffch-client.service.ts

```diff
@@ -63,6 +65,18 @@ export class FfchClientService {
     return this.http.post(FFCH_EQUIPMENTS_PATH, uniteEquipements);
   }
 
+  public postUniteCompetence$(uniteCompetence: UniteCompetence, uniteId: number): Observable<UniteCompetence> {
+    let params = new HttpParams();
+    params = params.append('uniteId', `${uniteId}`);
+    return this.http.post<UniteCompetence>(FFCH_UNITE_SKILLS_PATH, uniteCompetence, {params: params})
```

Comment:
La bonne syntaxe est `return this.http.post<UniteCompetence>(FFCH_UNITE_SKILLS_PATH, uniteCompetence, {params})` pour cette ligne.